docker-compose.test.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  web:
+    build:
+      context: .
+      target: test
+      args:
+        RAILS_ENV: test
+    image: local/dfe-teachers-payment-service:test
+    depends_on:
+      - db
+    env_file:
+      - .env.development
+    environment:
+      DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_USERNAME: postgres
+      DFE_TEACHERS_PAYMENT_SERVICE_DATABASE_HOST: db
+    tty: true
+    stdin_open: true
+    command: bundle exec spring server
+
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
```

Comment:
This means it's sharing a database with the `development` Docker compose setup. Was that intentional?

Reply:
Good catch, I'll change it. Azure automatically opened this PR when I went to enable pipelines so it's very much WIP, I'll fix it up